### DISCUSSION TITLE
[SDK] Can't bridge eth from L1 using the Python SDK and the example from the documentation

### DISCUSSION THREAD

**Discussion by 0xuser** [zkSync-Community-Hub/zksync-developers#55] [SDKs] [answered]
### Environment

Mainnet

### Acknowledgement

- [x] I understand that issues are expected with the Python and Java SDK as they are being updated.

### Issue Description

I can't bridge eth from L1 using the Python SDK and the example from the documentation.

I have funds for both bridge and the Ethereum transaction fee but I still get the following error:
`ValueError: {'code': -32000, 'message': 'insufficient funds for gas * price + value'}`

Please help. I saw on discord that more people had a problem with this however I didn't see any response.

### Expected Behavior

Bridge of ETH funds from the Ethereum network to the ZkSync Era network

### Code Example

Zksync Era docs example:
[https://era.zksync.io/docs/api/python/getting-started.html#deposit-eth](https://era.zksync.io/docs/api/python/getting-started.html#deposit-eth)

### Repo Link (Optional)

https://era.zksync.io/docs/api/python/getting-started.html#deposit-eth

**Comment by uF4No** [comment, marked as answer]
Issue identified. We're working on it and it will be fixed in the next version of the SDK

**Reply by baalisgood** [reply]
I met the same problem, when I used the test network, the code was successful, but when I switched to the main network, 'eth_provider.deposit' would generate a large unreasonable cost value, in fact, my account balance was satisfied. But this cost value is 33 times greater than the amount I need to use the front-end web cross-chain, which is exaggerated

**Reply by baalisgood** [reply]
How long will this error take to recover

**Comment by 0xuser** [comment]
Ok, thank you for the information. I thought my code was wrong but I figured out that the example from the documentation doesn't work correctly either.

**Reply by baalisgood** [reply]
Hello, have you solved this problem?

**Reply by 0xuser** [reply]
use js SDK

**Comment by amyliasnow** [comment]
It seems like the error "insufficient funds for gas * price + value" indicates that the provided account doesn't have enough funds to cover the gas fee for the transaction. Please double-check that the account you're using has sufficient funds not only for the bridge operation but also for the gas fee required by Ethereum. If the issue persists, consider adjusting the gas price or gas limit parameters accordingly. Additionally, ensure that you're using the correct account and network configurations in your Python SDK setup. If further assistance is needed, feel free to provide more details or reach out to the zkSync community for support.

**Comment by ryangrullon** [comment]
The error message "insufficient funds for gas * price + value" typically indicates that there are not enough funds available to cover the gas cost of the transaction in addition to the value being transferred. Make sure that you have enough ETH in your wallet to cover both the transaction fee and the amount you're trying to bridge to the ZkSync Era network. Additionally, ensure that you're specifying the gas price and gas limit correctly in your transaction parameters. If the issue persists, consider reaching out to the ZkSync support team for further assistance.

**Comment by reginaldolivas** [comment]
Ensure that you have provided sufficient funds to cover both the gas fees and the value being bridged. 
Double-check the gas price and the amount being bridged to ensure they are within the available balance.

**Comment by postm4ltolin** [comment]
The error message "insufficient funds for gas * price + value" indicates that the account used for the transaction does not have enough funds to cover the gas fee and value being sent. Double-check that you have enough funds in your account to cover both the gas fee and the value being sent. You may also want to check the gas price being used and adjust it if necessary. Additionally, make sure that you are using the correct network and that the account has been properly authenticated.

**Comment by cioprogephen** [comment]
The user is encountering an issue with bridging ETH from L1 to the ZkSync Era network using the Python SDK and the example from the documentation on the Mainnet. The user has funds for both the bridge and the Ethereum transaction fee but is still receiving the error message "insufficient funds for gas * price + value." The user has not provided any relevant portions of their code, but they are using the ZkSync Era docs example for depositing ETH. The user has acknowledged that issues are expected with the Python and Java SDK as they are being updated. No repo link is provided.
